MetaMask/metamask-extension
# fix: replace dnode background with JSON-RPC

Fixes #10090

Explanation:  This replaces our internal use of `dnode` with `JSON-RPC`. 

Manual testing steps:  
  - make sure everything still works, this is a cross-cutting change that affects most (if not all) of the application
